benchmark/parse_output_gi.py

```diff
@@ -0,0 +1,79 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015--, The WGS-HGT Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+#
+# Parse output files of GI tools.
+#
+
+import sys
+import click
+from skbio import Sequence
+
+
+def parse_output_gi(genbank_fp,
+                    gi_fp):
+    """ Identify genes contained in GIs
+
+    Parameters
+    ----------
+    genbank_fp: string
+        file path to genome in GenBank format
+    gi_fp: string
+        file path to GI list
+
+    Returns
+    -------
+    output: string
+        gene names (protein_ids) separated by newline
+    """
+    genes = {}
+    gb = Sequence.read(genbank_fp, format='genbank')
+    for feature in gb.interval_metadata.features:
+        if feature['type_'] == 'CDS':
+            if 'protein_id' in feature:
+                protein_id = feature['protein_id'].replace('\"', '')
+                loc = gb.interval_metadata.features[feature]
```

Comment:
I inherited this code from previous codes and honestly don't know why it is phrased like this. Besides, I noticed that these expressions are different in scikit-bio 0.4.2 versus 0.5.0. The latter looks more reasonable to me.